src/AndroidDebugLauncher/InstallPaths.cs

```diff
@@ -11,6 +11,7 @@
 using System.Threading;
 using System.Threading.Tasks;
 using System.Globalization;
+using MICore;
 
 namespace AndroidDebugLauncher
```

Comment:
The comments on GDBPath, and GDBServerPath is wrong with your change (they are now optional). Please fix.